README.md

```diff
@@ -152,12 +161,16 @@ function getAllAddons({ categoryId }: GetAllAddonsParams = {}) {
 
 ### Code coverage
 
-The `npm run unittest` command generates a report of how well the unit tests
-covered each line of source code.
-The continuous integration process will give you a link to view the report.
-To see this report while running tests locally, type:
+To see a report of code coverage, type:
+````
+yarn test-coverage
+````
 
-    open ./coverage/index.html
+This will print a table of files showing the percentage of code coverage.
+The uncovered lines will be shown in the right column but you can open
+the full report in a browser:
+
+    open coverage/lcov-report/index.html
```

Comment:
This only works on Mac OS, for what it's worth. Also might be nice to be consistent with block code style as the other ones use the tilde.